mage_ai/orchestration/notification/config.py

```diff
@@ -26,6 +27,7 @@ class NotificationConfig(BaseConfig):
     email_config: EmailConfig = None
     slack_config: SlackConfig = None
     teams_config: TeamsConfig = None
+    google_chat_config: GoogleChatConfig = None
```

Comment:
nit: same here. could you sort them alphabetically (put it below `email_config`)?